src/plugins/data/public/autocomplete/collectors/create_usage_collector.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { first } from 'rxjs/operators';
+import { StartServicesAccessor } from '../../../../../core/public';
+import { METRIC_TYPE, UsageCollectionSetup } from '../../../../usage_collection/public';
+import { AUTOCOMPLETE_EVENT_TYPE, AutocompleteUsageCollector } from './types';
+
+export const createUsageCollector = (
+  getStartServices: StartServicesAccessor,
+  usageCollection?: UsageCollectionSetup
+): AutocompleteUsageCollector => {
+  const getCurrentApp = async () => {
+    const [{ application }] = await getStartServices();
+    return application.currentAppId$.pipe(first()).toPromise();
+  };
+
+  return {
+    trackCall: async () => {
+      const currentApp = await getCurrentApp();
+      return usageCollection?.reportUiCounter(
+        currentApp!,
+        METRIC_TYPE.LOADED,
+        AUTOCOMPLETE_EVENT_TYPE.CALL
+      );
+    },
+    trackRequest: async () => {
+      const currentApp = await getCurrentApp();
+      return usageCollection?.reportUiCounter(
+        currentApp!,
+        METRIC_TYPE.LOADED,
```

Comment:
This is how this metrics are logged: 

![Screenshot 2021-02-16 at 16 27 42](https://user-images.githubusercontent.com/7784120/108083950-f215d880-7073-11eb-8adf-2c8d3643ebf6.png)

I think those are not specific enough